projects/questionset-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -153,6 +154,10 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
         this.getFrameworkDetails(this.primaryCategoryDef);
       }
     });
+    this.editorService.readComment(this.collectionId).subscribe((res) => {
```

Comment:
It would be good if you move the comments logic to different component or service which will handle only comments adding, updating and removing.

Reply:
sure, will check on that.

Reply:
I've moved the almost all the logics to service files
1. Read Comments not able to optimise more.
2. Updated comment all logics moved to editor service.